common/app/views/fragments/amp/stylesheets/fonts.scala.html

```diff
@@ -27,8 +26,6 @@
 .guardian-egyptian-loaded .fc-container__header__description,
 .guardian-egyptian-loaded .paid-content--advertisement-feature,
 .guardian-egyptian-loaded .witness__button,
-.guardian-egyptian-loaded .hosted-article-page,
```

Comment:
We used guardian egyptian at first, since we didn't have a guardian sans, but since it's commercial content we decided a sans serif font is better.

I haven't had time to look into this AMP font loading mechanism, probably will go into a different PR, this one already has some scope creep!